KPP/fullchem/gckpp_Function.F90

```diff
@@ -44,7 +44,7 @@ MODULE gckpp_Function
 !
 ! ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-SUBROUTINE Fun ( V, F, RCT, Vdot, Aout )
+SUBROUTINE Fun ( V, F, RCT, Vdot, Aout, Vdotout )
```

Comment:
Add this option to the standard model?

Reply:
I added this for the NOxTau diagnostics. Emma is still testing this feature, but I added them to my main development branch because I wanted to make sure that the updates are being passed along. It's a zero diff update so I don't see why it would hurt to include it. We could bracket the NoxTau diagnostics in MODEL_GEOS brackets to highlight that it is not yet fully tested.

Reply:
Sounds good.